hamocc/mo_param_bgc.F90

```diff
@@ -148,6 +155,10 @@ module mo_param_bgc
   real, protected :: bkphy      = 4.e-8           ! kmol/m3 - i.e. 0.04 mmol P/m3 half saturation constant
   real, protected :: dyphy      = 0.004           ! 1/d -mortality rate of phytoplankton
 
+  ! Initial fractionation during photosynthesis
```

Comment:
In the comment for this initialization section, I would suggest to add DMS information (i.e. `Phytoplankton parameters (incl. cyanobacteria and -partially microbial- DMS production`) or something similar.

Reply:
I have made an extra header for DMS and BrO schemes